examples/official-storybook/stories/core.stories.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+storiesOf('core/options', module).add(
+  'passed to story',
+  ({ options: { myOption } }) => (
+    <div>
+      Value of <code>myOption</code> is '<code>{myOption}</code>'
+    </div>
+  ),
+  { myOption: 'value' }
+);
```

Comment:
See this example for insight into how this feature works. More importantly, decorators and addons get passed the `context` also.